src/autocomplete.js

```diff
@@ -108,6 +123,10 @@ class Autocomplete {
         return this.inlineRenderer;
     }
 
+    /**
+     * 
+     * @return {AcePopup}
```

Comment:
why is this not inferred? Also why do you include the empty lines here?

Reply:
Providing implicit type here allows typescript compiler not to go through inside block code to get return type. Empty line was just auto-generated

Reply:
It also helps doc generators while providing a quick error once a rewrite causes an unexpected inference, though some people drop `@returns {void}` anyway